src/nanovg.c

```diff
@@ -2416,8 +2416,21 @@ static int nvg__allocTextAtlas(NVGcontext* ctx)
 {
 	int iw, ih;
 	nvg__flushTextTexture(ctx);
-	if (ctx->fontImageIdx >= NVG_MAX_FONTIMAGES-1)
-		return 0;
+	if (ctx->fontImageIdx >= NVG_MAX_FONTIMAGES-1) {
+		int i;
```

Comment:
I wonder if we should call render flush here before deleting the images? I think now it's possible that because of the caching, there might be some render calls in flight when we delete the textures. That's the reason why the deletes happen at `nvgEndFrame()`.